src/common/localization/en-us.ts

```diff
@@ -334,6 +334,9 @@ export const english: IAppStrings = {
             notCompatibleWithDrawnRegionTag: "Drawn regions and ${otherCategory} values cannot both be assigned to the same document's tag",
             replaceAllExitingLabels: "Are you sure you want to replace selected tag's labels?",
             replaceAllExitingLabelsTitle: "Replace tag's labels",
+            imageTagsOnlySupportDrawRegion:"Image tags only support Draw region",
```

Comment:
please use "signatureTagsOnlySupportDrawRegion", and the same logic applies to other strings.